infra/indexer/Pulumi.staging.yaml

```diff
@@ -0,0 +1,17 @@
+secretsprovider: awskms:///arn:aws:kms:us-west-2:968153779208:alias/pulumi-secrets-key
+encryptedkey: AQICAHhnm/1/W7/xeF2uxmXOGjFzOf6jjMX+KgbMb0K+LSCbsAHtr8teNb0/7Lg/Tm/X0vsDAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM25ECvR4QS8hDmELeAgEQgDvCxRxFXZ2CR5qk3YJ7/30fSy77uQQl1rwBT3P3Tx0DzRTuQUMx14qEWGt/k3CB8O8EOp7Xw2CehaFs3g==
+config:
+  aws:region: us-west-2
+  indexer:BASE_STACK: organization/bootstrap/services-staging
+  indexer:BOUNDLESS_ADDRESS: 0x2d124b86179F0De2d5B7A75E4dcD7e35a3363B16
+  indexer:DOCKER_DIR: ../../
+  indexer:DOCKER_TAG: latest
+  indexer:START_BLOCK: 8038641
+  indexer:ETH_RPC_URL:
+    secure: v1:D0LPseptkWMnYnOv:wW8eWtEA5HHwiJ7xc4RJIHWHCpEmcSAq8OHMeMk0HSYnZpLQXMuax6MqgFQBIzlNWkJKTCTaBw98Oj6TfUvnA8hC0Ub9NJejUHnJdzgKrEL3tscP7g==
+  indexer:RDS_PASSWORD:
+    secure: v1:Q8vUXQ59MCkcmyw8:C8BUe4hjIDn+biTuqdDYoluH+rM6eRMO0LKjjmpN9EpQ7A==
+  indexer:CHAIN_ID: "11155111"
+  indexer:SLACK_ALERTS_TOPIC_ARN: arn:aws:sns:us-west-2:968153779208:boundless-alerts-topic
+  indexer:CLIENTS: "0xe9669e8fe06aa27d3ed5d85a33453987c80bbdc3"
+  indexer:PROVERS: "0x6bf69b603e9e655068e683bbffe285ea34e0f802,0xf8087e8f3ba5fc4865eda2fcd3c05846982da136"
```

Comment:
We're going to want to set specific alarms per client/prover, likely with different thresholds etc. I'm inclined to keep all the addresses and their alarm config in a config file (I would just make it a `.ts` file with a json object)